pkg/operator/csi/csicontrollerset/csi_controller_set.go

```diff
@@ -110,6 +111,7 @@ func (c *CSIControllerSet) WithCredentialsRequestController(
 	assetFunc resourceapply.AssetFunc,
 	file string,
 	dynamicClient dynamic.Interface,
+	typedVersionedOperatorClient operatorv1client.Interface,
```

Comment:
except efs operator, I did not find a single use of this function in any other openshift project and hence I figured breaking this function interface should be an acceptable tradeoff. cc @bertinatto

Reply:
Yeah, I think this controller is not used anywhere else.